hooks/pre_commit/42-evr_extension_permissions.rb

```diff
@@ -0,0 +1,30 @@
+# Managed databases will be handled automatically.
+return if local_postgresql?
+
+database = param_value('foreman', 'db_database') || 'foreman'
+username = param_value('foreman', 'db_username') || 'foreman'
+password = param_value('foreman', 'db_password')
+host = param_value('foreman', 'db_host')
+port = param_value('foreman', 'db_port') || 5432
+
+# If postgres is the owner of the DB, then the permissions will not matter.
+return if username == 'postgres'
+
+check_evr_owner_sql = "SELECT CASE" \
+                      " WHEN r.rolname = 'postgres' THEN 1" \
```

Comment:
Wouldn't it make more sense to verify that the owner equals the username to connect to the DB?

Reply:
Yeah, and that way we wouldn't have to assume that postgres is the default admin. I'll try it out.